plugin-base/src/main/java/io/everytrade/server/parser/exchange/XChangeApiTransaction.java

```diff
@@ -39,6 +39,7 @@ public class XChangeApiTransaction {
     BigDecimal price;
     BigDecimal feeAmount;
     String address;
+    Boolean isBittrexApiDepositTx;
```

Comment:
Please rename isBittrexApiDepositTx to logIgnoredFees

Reply:
should be true by default

Reply:
I don't know how to set the attribute "true" by default (simplyfy inicialization "logIgnoredFees = true" doesn't work here),  but it works fine after changing the condition.